### ISSUE TITLE
F-Droid build failed

### ISSUE THREAD

**Issue by linsui** [Mosc/Glider#34]
```
Codepoint 62188 not found in font, aborting.
FAILURE: Build failed with an exception.
* Where:
Script '/home/vagrant/build/srclib/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 1005
* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command '/home/vagrant/build/srclib/flutter/bin/flutter'' finished with non-zero exit value 1
```

Could you please take a look? Thanks!

**Comment by Mosc** [comment]
Thanks for the heads up. I believe this is the same issue that was fixed for the Play Store build in 9768e6bfebbaddc8068bac4c83dcfaf9845f0636. Not sure why tree shaking suddenly breaks the build, but I'll submit the same workaround for F-Droid.

**Comment by linsui** [comment]
Thanks!
